cupy/testing/_pytest_impl.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+import cupy.testing.parameterized
+
+try:
+    import pytest
+    _error = None
+except ImportError as e:
+    _error = e
+
+
+def is_available():
+    return _error is None
+
+
+def check_available():
+    if _error is not None:
+        raise RuntimeError('''\
+{} is not available.
+
+Reason: {}: {}'''.format(__name__, type(_error).__name__, _error))
+
+
+if is_available():
+
+    class _TestingParameterizeMixin:
+
+        def __repr__(self):
+            return "<{}  parameter: {}>".format(
```

Comment:
Please use single quotes instead.

Reply:
Sure.  Done in a6e02a63c16c8e63e74923efb4bca84ef62a227b.